homeworks/src/homework6/ArrayValueCalculator.java

```diff
@@ -0,0 +1,59 @@
+package homework6;
+
+public class ArrayValueCalculator {
+
+    public static void main(String[] args) {
+
+        String[][] arr = {
+                {"1", "998", "567", "1234"},
+                {"45", "998", "1", "123345"},
+                {"45", "998", "567", "1234"},
+                {"45", "998", "567", "456"}
+        };
+
+        int res = 0;
+        boolean err = false;
+
+        try {
+            res = doCalc(arr);
+        } catch (ArraySizeException ex) {
+            System.out.println(ArraySizeException.getErrorMessage());
+            err = true;
+        } catch (ArrayDataException ex) {
+            System.out.println(ArrayDataException.getErrorMessage());
```

Comment:
With the same catch bodies, you may use  `catch (ArraySizeException|ArrayDataException  ex)`